core/auth/src/main/java/software/amazon/awssdk/auth/credentials/internal/ProfileCredentialsUtils.java

```diff
@@ -83,7 +83,8 @@ public ProfileCredentialsUtils(Profile profile, Function<String, Optional<Profil
      * providers will be created.
      */
     public Optional<AwsCredentialsProvider> credentialsProvider() {
-        return credentialsProvider(new HashSet<>());
+        Optional<AwsCredentialsProvider> credsProvider = credentialsProvider(new HashSet<>());
+        return credsProvider;
```

Comment:
Why the change?

Reply:
Not sure, might have been me testing and missed reverting.